build_tools/gradle/build_android.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Cross-compile the IREE's TFLite Java Bindings with Gradle and CMake.
```

Comment:
Please be more specific with this file name here - there is already [another `build_android.sh` script](https://github.com/google/iree/blob/main/build_tools/cmake/build_android.sh) and we should differentiate them.

also nit: remove 'the' from this comment
```suggestion
# Cross-compile IREE's TFLite Java Bindings with Gradle and CMake.
```